main

```diff
@@ -1,44 +1,19 @@
 #!/bin/bash
 
-
 #clean up previous job (just in case)
 rm -f finished
 
-
-#if [ $ENV == "IUHPC" ]; then
-#	module unload python
-#	if [ $HPC == "KARST" ]; then
-#		module load anaconda2
-#	fi
-#	if [ $HPC == "CARBONATE" ]; then
-#		module load anaconda/python2.7/4.3.1
-#	fi
-	
-#fi
-	
-	#module unload python/2.7.3
-
-module load singularity 2> /dev/null
```

Comment:
I'm not sure this should be removed, I think it will not work on my carbonate without